lms/templates/dates_banner.html

```diff
@@ -52,7 +52,7 @@
         </div>
         % if not is_mobile_app:
             <div class="upgrade-button">
-                <a href="${verified_upgrade_link}">
+                <a id="PLS_button" href="${verified_upgrade_link}">
```

Comment:
IDs should be unique per element per page, so you'll probably want to use a class instead.

Reply:
There will be only one of these displayed on a page at a time; they're both defined up top (as "upgrade_to_reset_banner" and "upgrade_to_complete_graded_banner"), and then the block at the bottom of the page has logic to display one or the other, so we'll never have both displayed at once- here's that section:
https://github.com/edx/edx-platform/blob/7edd95dfe4d4042c7e65928f4f6ad9b80c28df6d/lms/templates/dates_banner.html#L89

Reply:
We discussed, and only one link can appear at a time right now, but I switched it to class instead of id to help future-proof it!